ATM_Selenium_Training/src/test/java/com/epam/atm/selenium/NewAppTest.java

```diff
@@ -0,0 +1,72 @@
+package com.epam.atm.selenium;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.regex.Pattern;
+
+public class NewAppTest extends BaseTest {
+
+    @Test
+    public void newApplication() {
+        int x = (int) (Math.random() * 1000);
+        driver.get("https://network-international--qa.cs83.my.salesforce.com/");
+        System.out.println("Open application (url) for testing");
+        Assert.assertEquals(driver.getTitle(), "Login | Salesforce", "Incorrect page (login page) is not opened");
+        System.out.println("Check that correct Login page is opened");
+        WebElement username = driver.findElementByXPath("//input[@class='input r4 wide mb16 mt8 username']");
+        username.sendKeys("yauhen_valodzin@epam.com.qa");
+        System.out.println("Find and populate Username field");
+        WebElement password = driver.findElementByXPath("//input[@class='input r4 wide mb16 mt8 password']");
+        password.sendKeys("Ceakt_1234");
+        System.out.println("Find and populate Password field");
+        WebElement button = driver.findElementByXPath("//input[@class='button r4 wide primary']");
+        button.click();
+        System.out.println("Find and click Login button");
+        WebElement dynamicElement = (new WebDriverWait(driver, 10).until(ExpectedConditions.presenceOfElementLocated((By.xpath("//span[text()='Recently Viewed']")))));
```

Comment:
do we need to save it in dynamicElement. It will be not used later

Reply:
Done